topics/outcomes.yaml

```diff
@@ -6,7 +6,6 @@ services:
     - getsentry/sentry
     - getsentry/relay
   consumers:
-    - getsentry/sentry
```

Comment:
I assume this was a typo, we have no consumer in sentry for outcomes

Reply:
there is one in getsentry

Reply:
ah ok